.github/actions/build_node/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10-slim
+FROM node:20-slim
```

Comment:
https://hub.docker.com/_/node/

I wasn't sure which one to change to 🙈 Feel free to update if there is an image that suits/works better!